Mathlib/MeasureTheory/Group/Integral.lean

```diff
@@ -48,7 +48,7 @@ variable [Group G] [MeasurableMul G]
 left-invariant measure. -/
 @[to_additive
       "Translating a function by left-addition does not change its integral with respect to a
-      left-invariant measure."] -- Porting note: was `@[simp]`
+      left-invariant measure."]
```

Comment:
Why are you removing the note: can simp prove this now? Same for the other occurrences.

Reply:
I'm removing it because no one has been bothered by the fact that it is isn't `@[simp]` for amost 2 years, so it apparently doesn't need to be!